cou/utils/juju_utils.py

```diff
@@ -330,7 +330,7 @@ async def _get_supported_apps(self) -> list[str]:
             name for name, app in model.applications.items() if is_charm_supported(app.charm_name)
         ]
 
-    async def _get_unit(self, name: str) -> JujuUnit:
+    async def get_unit(self, name: str) -> JujuUnit:
```

Comment:
These `_get_unit` -> `get_unit` do not seem to be required here, and it's not used outside `Model` class

Reply:
They'll be used when adding the vault unseal feature next. :)

Reply:
Yes